modules/function.js

```diff
@@ -0,0 +1,7 @@
+import timeAndDate from './variables.js';
+import { DateTime } from './luxon.js';
+
+export default function dateAndTimeNow() {
+  const date = DateTime.now().toLocaleString(DateTime.DATETIME_FULL_WITH_SECONDS);
+  timeAndDate.innerHTML = date;
+}
```

Comment:
- [ ] Kindly change the functions to arrow functions since this is a project requirement.

<img width="455" alt="image" src="https://user-images.githubusercontent.com/61919617/193860924-0f07d450-700d-42a3-838a-b4040dbd48d9.png">